src/components/form-select/form-select.types.ts

```diff
@@ -0,0 +1,6 @@
+import { SelectProps } from "antd";
+
+export type FormSelectProps = SelectProps & {
+    name: string;
+    label?: string;
+};
```

Comment:
we just previously used "extends ComponentProps&lt;typeof Input&gt;" notation when writing reusable input component types